Simple Inventory Management System/Program.cs

```diff
@@ -8,7 +8,7 @@ class Program
     static async Task Main(string[] args)
     {
         var config = new ConfigurationBuilder()
-            .AddJsonFile(@"appsettings.json", optional: false, reloadOnChange: true)
+            .AddJsonFile("C:\\Users\\hp\\Desktop\\New folder\\Simple Inventory Management System\\appsettings.json", optional: false, reloadOnChange: true)
```

Comment:
why? my question is why you have used "@"

Reply:
both sides true! , I am not add "\"

Reply:
My meaning is :
```csharp
.AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
```